vocab/vocab.csv

```diff
@@ -1234,6 +1239,14 @@ GI05MOAS-PG564-03-FLORTM000,Global Irminger Sea,Mobile Assets,Profiling Glider 5
 GI05MOAS-PG564-04-FLORTO000,Global Irminger Sea,Mobile Assets,Profiling Glider 564,3-Wavelength Fluorometer,WET Labs,ECO BB3,0,200
 GI05MOAS-PG564-05-NUTNRM000,Global Irminger Sea,Mobile Assets,Profiling Glider 564,Nitrate,Satlantic,SUNA V2,0,200
 GI05MOAS-PG564-06-PARADM000,Global Irminger Sea,Mobile Assets,Profiling Glider 564,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,200
+GI05MOAS-PG565,Global Irminger Sea,Mobile Assets,Profiling Glider 565,,,,0,200
+GI05MOAS-PG565-00-ENG000000,Global Irminger Sea,Mobile Assets,Profiling Glider 565,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,200
+GI05MOAS-PG565-01-CTDGVM000,Global Irminger Sea,Mobile Assets,Profiling Glider 565,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,200
+GI05MOAS-PG565-02-DOSTAM000,Global Irminger Sea,Mobile Assets,Profiling Glider 565,Dissolved Oxygen,Aanderaa,Optode 4831,0,200
+GI05MOAS-PG565-03-FLORTM000,Global Irminger Sea,Mobile Assets,Profiling Glider 565,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,200
```

Comment:
Reminder that we need to clean up 'vocab' at some point to correctly list the model of FLORTM as FLBBCDSLC (or list both models if possible since originally we did also have FLBBCD-SLK at one point...both have same 'M' series).